src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -1167,4 +1177,42 @@ private String buildQueryWithRequesterId(String requesterId, String sortBy) {
     return format("(requesterId==%s and status==Open*)", requesterId);
   }
 
+  private CompletableFuture<Boolean> isEcsTlrFeatureEnabled(VertxOkapiHttpClient httpClient,
+    Map<String, String> okapiHeaders) {
+
+    return httpClient.get(ECS_TLR_SETTINGS_URL_PATH.getValue(), Map.of(), okapiHeaders)
+      .thenApply(ResponseInterpreter::extractResponseBody)
```

Comment:
Why not use existing method `verifyAndExtractBody`?

Reply:
fixed